### DISCUSSION TITLE
Add preview page to see the effect of message template

### DISCUSSION THREAD

**Discussion by yeahCaesar** [grafana/grafana#59779] [Alerting]
It is so hard to debug the message template.I need to send an email every time once I modify the message template.Could Grafana provide a page to preview the message template - how the message will be exactly with data.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
